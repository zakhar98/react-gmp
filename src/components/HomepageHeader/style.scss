.homepage-header {
  height: 400px;
  background: radial-gradient(rgba(0, 0, 0, 0.8), rgba(0, 0, 0, 0.8)), url(https://assets.nflxext.com/ffe/siteui/vlv3/e178a4e7-4f52-4661-b2ae-41efa25dca7c/eb618394-24cd-43ae-8799-c1a76a7bcbfc/BY-en-20210222-popsignuptwoweeks-perspective_alpha_website_small.jpg);
  padding: 24px 72px;

  &__top {
    display: flex;
    justify-content: space-between;
    margin-bottom: 40px;
  }

  &__add-movie-button {
    color: #f65261;
    font-size: 25px;
    background-color: rgba(255, 255, 255, .2);
    width: 220px;
    border-radius: 4px;
    padding: 10px;
    text-align: center;
    border: none;
    text-transform: uppercase;
  }

  &__content {
    margin: 0px 80px;
  }

  &__content-text {
    color: white;
    font-size: 50px;
    font-weight: 200;
    margin-bottom: 24px;
  }
}
