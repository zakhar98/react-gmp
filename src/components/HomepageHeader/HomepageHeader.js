import React from "react";
import Logo from "../Logo/Logo.js";
import SearchBar from "../SearchBar/SearchBar.js";
import "./style.scss";

export default function HomepageHeader() {
  return (
    <header className="homepage-header">
      <div className="homepage-header__top">
        <Logo />
        <button className="homepage-header__add-movie-button"> + add movie</button>
      </div>
      <div className="homepage-header__content">
        <div className="homepage-header__content-text">FIND YOUR MOVIE</div>
        <SearchBar />
      </div>
    </header>
  );
}
