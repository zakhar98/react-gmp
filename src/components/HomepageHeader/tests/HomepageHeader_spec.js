import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import HomepageHeader from '../HomepageHeader.js';

describe('HomepageHeader', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer
      .render(<HomepageHeader />);
    expect(tree).toMatchSnapshot();
  });
});
