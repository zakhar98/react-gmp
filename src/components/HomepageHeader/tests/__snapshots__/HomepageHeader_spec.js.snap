// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageHeader renders correctly 1`] = `
<header
  className="homepage-header"
>
  <div
    className="homepage-header__top"
  >
    <Logo />
    <AddMovieButton />
  </div>
  <div
    className="homepage-header__content"
  >
    <div
      className="homepage-header__content-text"
    >
      FIND YOUR MOVIE
    </div>
    <Memo(Connect(withRouter(SearchBar))) />
  </div>
</header>
`;
